src/apps/weblib/ts-api/data.ts

```diff
@@ -130,6 +165,30 @@ export class Data {
 		return true
 	}
 
+	private _convertDimension(values: string[] | (number | null)[]): { categories: string[], indexes: number[] } {
```

Comment:
The type hint probably is the following: `(string | number | null)[]`

Reply:
Can this Values type hint be used here?

Reply:
Yes, but it's better to revise the Data types altogether to represent better the intended API. We discussed it with @dovicsin and he is working on it.